demo_grid/js/demo/index.js

```diff
@@ -17,6 +17,10 @@
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License version 3.0
  */
 
+// Since PrestaShop 1.7.8 you can import belows component directly from the window object.
```

Comment:
```suggestion
// Since PrestaShop 1.7.8 you can import global components directly from the window object.
```
is "bellows' a word in this context? (not an expert, but sounds very strange :smile: so I suggest this to be safe)